frontend/src/App.tsx

```diff
@@ -141,34 +145,40 @@ function App() {
         <div className={styles.Content}>
           <Router>
             <ScrollToTop />
-            <AppBar position="static" elevation={0}>
-              {width > MOBILE_BREAKPOINT ? (
-                <AppToolbar />
-              ) : (
-                <MobileAppToolbar />
-              )}
-            </AppBar>
-            <Switch>
-              <Route path={ABOUT_US_PAGE_LINK} component={AboutUsPage} />
-              <Route path={DATA_CATALOG_PAGE_LINK} component={DataCatalogTab} />
-              <Route
-                path={EXPLORE_DATA_PAGE_LINK}
-                component={ExploreDataPage}
-              />
-              <Route
-                path={WHAT_IS_HEALTH_EQUITY_PAGE_LINK}
-                component={WhatIsHealthEquityPage}
-              />
-              <Route
-                  path={TERMS_OF_SERVICE_PAGE_LINK}
-                  component={TermsOfServicePage}
-              />
-              <Route exact path="/" component={LandingPage} />
-              <Route component={NotFoundPage} />
-            </Switch>
+            <header aria-label="Header content including site navigation">
```

Comment:
I do not think you need an aria-label here. Having <header> provides the page structure, and as long as the navigation links area announced, I think that is all that needs to be announced.